description: >
    Lane filter for estimating the distance to the stop line
constructor: lane_filter_generic.LaneFilterMoreGeneric
parameters:

    configuration:
        variables: !!omap
            - d:
                min: -0.15
                max: +0.3025

                resolution: 0.01
                # resolution: 0.02 # like baseline

                units: m
                units_display: cm
                description: Distance from center lane
            - dstop:
                # min: 0.0
                min: -0.1
                max: 0.5
                resolution: 0.01
                # resolution: 0.02 # like baseline

                units: m
                units_display: cm
                description: Distance from stop line


        localization_template: DT17_template_xy_stopline

        delta_segment: 0.0164

        F: 1
        optimize: true
        bounds_theta_deg: [-45, 45]
        precision: float32
